charts/website/templates/secret.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-discord
+data:
+  client-id: {{ .Values.siteConfig.discord.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.discord.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-google
+data:
+  client-id: {{ .Values.siteConfig.google.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.google.client-secret | b64enc }}
+  yt-api-key: {{ .Values.siteConfig.google.yt-api-key | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-github
+data:
+  client-id: {{ .Values.siteConfig.github.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.github.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-btlnet
+data:
+  client-id: {{ .Values.siteConfig.btlnet.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.btlnet.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-twitch
+data:
+  client-id: {{ .Values.siteConfig.twitch.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.twitch.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-facebook
+data:
+  client-id: {{ .Values.siteConfig.facebook.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.facebook.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-cloudflare
+data:
+  turnstile-secret: {{ .Values.siteConfig.cloudflare.turnstile-secret | b64enc }}
+  turnstile-site-key: {{ .Values.siteConfig.cloudflare.turnstile-site-key | b64enc }}
```

Comment:
add next_public here 2 ^ (if needed)

Reply:
turnstile secret is public?

Reply:
resolved in discord, only site key needs to be